forwards_paper.tex

```diff
@@ -63,7 +65,9 @@
     We implement the method in two popular forwards-time simulation frameworks,
     and show that not only does this speed up large simulations by one or two orders of magnitude,
     but saves substantially more information from the simulation
-    (all marginal genealogies).
+    (all marginal genealogies) than simply recording the final sequences of
+    simulated individuals.
+    \jda{ this is still only 120 words and we should say 'more than what'}
```

Comment:
how about "but saves not only full genotype sequences, but also all marginal geneologies of the simulated individuals."